src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,15 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+
+        return numbers.stream()
+                .map(string -> string.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
```suggestion
                .flatMap(n -> Arrays.stream(n.split(",")))
```
it's only a recommendation (we can use one method instead of two)

Reply:
wow
But itn't it's reads better than 1 line?